src/common/commonutils/CommonUtils.h

```diff
@@ -56,6 +56,8 @@ int RenameFile(const char* original, const char* target, OsConfigLogHandle log);
 typedef int(*CommandCallback)(void* context);
 
 int ExecuteCommand(void* context, const char* command, bool replaceEol, bool forJson, unsigned int maxTextResultBytes, unsigned int timeoutSeconds, char** textResult, CommandCallback callback, OsConfigLogHandle log);
+void AddMockCommand(const char* expectedCommand, bool matchPrefix, const char* output, int returnCode);
+void CleanupMockCommands();
 
```

Comment:
Can you please move these to the commonutils library unit-tests? Such mocking functions should be not part of the prod code.